operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -45,26 +47,30 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
       ```
 
   > **NOTE:** All subsequent steps of this procedure should be performed within the `/root/site-init` directory created in this step.
-   
-2. Update the LDAP settings in the customizations.yaml file.
 
-   1. (Optional) Add the LDAP CA certificate in the certs.jks section of customizations.yaml.
+2. Repopulate the keycloak_users_localize and cray-keycloak Sealed Secrets in the customizations.yaml file with the desired configuration.
 
-      Follow step 8 under the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure.
-   
-   2. Update the LDAP settings.
-  
-      LDAP connection information is stored in the keycloak-users-localize Secret in the services namespace. 
-      In the customizations.yaml file, set the values for the keycloak_users_localize keys in the spec.kubernetes.sealed_secrets field:
+   Update the LDAP settings with the desired configuration. LDAP connection information 
+   is stored in the keycloak-users-localize Secret in the services
+   namespace in the customizations.yaml file.
```

Comment:
There's no namespaces in the customizations.yaml file. I suggest change
"keycloak-users-localize Secret in the services
   namespace in the customizations.yaml file."

to

"keycloak_users_localize Sealed Secret in the customizations.yaml file."